tests/fsharp/Compiler/Conformance/BasicGrammarElements/BasicConstants.fs

```diff
@@ -9,8 +9,10 @@ module ``Basic Grammar Element Constants`` =
 
     [<Fact>]
     let ``Basic constants compile `` () =
+        let tripleQuotedByteString = " \"\"\" \" \"\"\"B "
+
         CompilerAssert.Pass 
-            """
+            $"""
 let sbyteConst = 1y
```

Comment:
I'm not sure if it's actually possible to escape triple quotes in a triple quoted string, so this'll have to do.